src/main/java/com/odysseusinc/athena/model/common/AthenaConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.odysseusinc.athena.model.common;
+
+public class AthenaConstants {
+
+    private AthenaConstants() {
+    }
+
+    public static final String VOCABULARY_RELEASE_VERSION_ID = "None";
```

Comment:
Hmm, just not to misguide let's change the name of constant
It's not a vocabulary release id, it's OMOP Vocabulary ID